### ISSUE TITLE
CUPS addon not installing: error 400

### ISSUE THREAD

**Issue by kskouzmin** [MaxWinterstein/homeassistant-addons#243]
Just try to install **CUPS** add-on https://github.com/MaxWinterstein/homeassistant-addons/tree/main/cups

My steps:
1. Add addon repository "https://github.com/MaxWinterstein/homeassistant-addons/" by Three Dots Menu.
2. Click to install button.
3. Get "**Unknown error, see supervisor logs**"
4. Get in logs: 
```
2024-03-11 14:42:22.177 ERROR (MainThread) [homeassistant.components.hassio.handler] /addons/1e14b3fb_cups/install return code 400
2024-03-11 14:42:22.178 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /addons/1e14b3fb_cups/install - 
```
5. Check for another add-ons from _this_ and other repos --- its installed an working as expect.

Please explain: it is out-of-date code in add-on (due to experimental status, last edit 4 months ago) or my system isn't correct? Or I am installing is wrong?

HA OS 12.0
HA Core: 2024.3.0
Hardware: Raspberry Pi 3, **armv7**

**Comment by kskouzmin** [comment]
@MaxWinterstein, as I understand this is because this add-on for armv7 is not a 'public'? So can you please set it?

**Comment by MaxWinterstein** [comment]
@kskouzmin good catch! Indeed there seems to be no image built for `armv7`

Comparing

https://github.com/MaxWinterstein/homeassistant-addons/pkgs/container/homeassistant-addon-cups-amd64

to 

https://github.com/MaxWinterstein/homeassistant-addons/pkgs/container/homeassistant-addon-cups-armv7

Will see what happened there, hopefully just some github action hiccup. Currently running low on time 😞

**Comment by kskouzmin** [comment]
Yes, amd64 and aarch64 is 3.0.3, but armv7 is 2.2.0. How it is working? GitHub must automatically generate it?

**Comment by MaxWinterstein** [comment]
If I remember correctly, it had issues to build on `armv7`. Some big addons already decided to ditch it, as it os mostly used on some Raspberry Pi 3, which is not really supported anymore. 

In fact, it is a slow machine with only 1GB of memory i think.

**Comment by kskouzmin** [comment]
Today i'm build modified version of this addon. **Just type 'armv7**' in config.yaml and build.yaml, point my repo to push image and **built it with official [HA Build Tool](https://github.com/home-assistant/builder)**. 

**All is fine:** built, pushed, pulled (**install by "Add-on Store"**), run and working with my Samsung ML-2160 on Raspberry Pi 3 B+. Now it shares this printer to linux, mac, android, ios. Nothing about errors or complex settings, GUI only.

**Comment by MaxWinterstein** [comment]
@kskouzmin in fact, it seems to build fine: https://github.com/MaxWinterstein/homeassistant-addons/actions/runs/8269881343/job/22626156568

thanks for the research and pointing out, will release in a few minutes

**Comment by MaxWinterstein** [comment]
@kskouzmin I released the armv7 version. Give it a few minuntes to build, and test if possible :)

**Comment by MaxWinterstein** [comment]
As time goes by, I dropped armv7 from my repo. There is simply not enough usage for it, and i simply don't have enough time. Sorry.
